openslides/utils/utils.py

```diff
@@ -29,3 +35,43 @@ def to_roman(number: int) -> str:
         return roman.toRoman(number)
     except (roman.NotIntegerError, roman.OutOfRangeError):
         return str(number)
+
+
+def get_element_id(collection_string: str, id: int) -> str:
+    """
+    Returns a combined string from the collection_string and an id.
+    """
+    return "{}:{}".format(collection_string, id)
+
+
+def split_element_id(element_id: Union[str, bytes]) -> Tuple[str, int]:
+    """
+    Splits a combined element_id into the collection_string and the id.
+    """
+    if isinstance(element_id, bytes):
+        element_id = element_id.decode()
+    collection_str, id = element_id.rsplit(":", 1)
+    return (collection_str, int(id))
+
+
+def get_user_id(user: Optional['CollectionElement']) -> int:
+    """
+    Returns the user id for an CollectionElement user.
+
+    Returns 0 for anonymous.
+    """
+    if user is None:
+        user_id = 0
+    else:
+        user_id = user.id
+    return user_id
+
+
+def str_dict_to_bytes(str_dict: Dict[str, str]) -> Dict[bytes, bytes]:
+    """
+    Converts the key and the value from a dict from st to bytes.
```

Comment:
Rewrite: `Converts the key and the value of a dict from str to bytes.`

Reply:
done